components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/RoleManagementService.java

```diff
@@ -451,6 +451,16 @@ Map<String, String> getMainRoleToSharedRoleMappingsBySubOrg(List<String> roleIds
     List<String> getAssociatedApplicationByRoleId(String roleId, String tenantDomain)
             throws IdentityRoleManagementException;
 
+    /**
```

Comment:
Update license header. 

Check other places as well.

Reply:
Addressed and resolved in commit: [42705b9](https://github.com/wso2/carbon-identity-framework/pull/6162/commits/42705b984942a14ec734891444d31c9d33397a6f)